bitnami/kafka/templates/svc-external-access.yaml

```diff
@@ -19,28 +19,23 @@ metadata:
 spec:
   type: {{ $root.Values.externalAccess.service.type }}
   {{- if eq $root.Values.externalAccess.service.type "LoadBalancer" }}
-  {{- if not (empty $root.Values.externalAccess.service.loadBalancerIPs) }}
+  {{- if $root.Values.externalAccess.service.loadBalancerIPs }}
   loadBalancerIP: {{ index $root.Values.externalAccess.service.loadBalancerIPs $i }}
   {{- end }}
   {{- if $root.Values.externalAccess.service.loadBalancerSourceRanges }}
   loadBalancerSourceRanges: {{- toYaml $root.Values.service.loadBalancerSourceRanges | nindent 4 }}
   {{- end }}
   {{- end }}
   ports:
-    - name: tcp-kafka
+    - name: kafka
```

Comment:
The new name doesn't follow [Istio standards](https://istio.io/docs/ops/deployment/requirements/), please revert the change to coninue using `tcp-kafka`.